src/extensions/program_test.rs

```diff
@@ -358,4 +374,16 @@ impl ProgramTestExtension for ProgramTest {
             );
         }
     }
+
+    #[cfg(feature = "pyth")]
+    /// Adds a Pyth oracle to the test environment.
+    fn add_pyth_oracle(
+        &mut self,
+        oracle: Pubkey,
+        program_id: Pubkey,
+        data: PriceAccount,
```

Comment:
Is is possible to simplify this? So users do not have to add whole PriceAccount? Probably 99% cases in tests we need to modify price only. So what about passing just arguments like price data and timestamp?

Reply:
Good point! I totally agree with you but I think it's good to have the option to provide the whole `PriceAccount` in a scenario it's ever needed.

I propose one of the following workarounds:

1. Create a separate function `add_pyth_oracle_simple` where you just provide the price data and timestamp
2. Change `add_pyth_orcale` arguments to 

```
fn add_pyth_oracle(
        &mut self,
        oracle: Pubkey,
        program_id: Pubkey,
        price_account: Option<PriceAccount>,
        price_info: Option<PriceInfo>,
        prev_timestamp: Option<i64>,
        ... other time options or etc
```

If you provide the full `price_account` the other arguments are ignored.
If you provide any of the other arguments those values are used and any missing values are set to the default.

I'm leaning to option 1 as it would be easier for new users. What do you think?

Reply:
I think that additional function is not necessary. 

Why not doing it in same function? Just do it as optionals. User can provide `price_account` or `price_info`. Then you can check if `price_account` is something then ignore `price_info`. But if user provide `price_info` only - use it and create recreate `PriceAccount` with defaults and this `price_info`